cocos/2d/CCLabel.cpp

```diff
@@ -1405,7 +1405,7 @@ void Label::onDraw(const Mat4& transform, bool transformUpdated)
             //draw text without outline
             glprogram->setUniformLocationWith4f(_uniformEffectColor,
                 _effectColorF.r, _effectColorF.g, _effectColorF.b, 0.f);
-            break;
```

Comment:
@zilongshanren @WenhaiLin 

 glprogram->setUniformLocationWith4f(_uniformTextColor,
     _textColorF.r, _textColorF.g, _textColorF.b, _textColorF.a);
 glprogram->setUniformLocationWith4f(_uniformEffectColor,
     _effectColorF.r, _effectColorF.g, _effectColorF.b, _effectColorF.a);
 for (auto&& batchNode : _batchNodes)
 {
     batchNode->getTextureAtlas()->drawQuads();
 }

//for the text with outline, the code above will draw the text part and the outline part

case LabelEffect::NORMAL:
    glprogram->setUniformLocationWith4f(_uniformTextColor,
        _textColorF.r, _textColorF.g, _textColorF.b, _textColorF.a);
    break;

//the text without outline will enter case LabelEffect::NORMAL, not case LabelEffect::OUTLINE, and will draw the text part using the code below

for (auto&& batchNode : _batchNodes)
{
    batchNode->getTextureAtlas()->drawQuads();
}

//so add return, both work OK, I have test it